portfolio/src/main/java/com/whz/portfolio/resource/StockQuoteSubscriber.java

```diff
@@ -89,4 +94,16 @@ public StockQuoteData get(String symbol) {
 
     return result;
   }
+
+  /**
+   * Helper method to receive an environment variable.
+   *
+   * @param key
+   * @param alt
+   * @return The environment variable. If not found the alt param is returned.
+   */
+  private String getenv(String key, String alt) {
+    String result = System.getenv(key);
+    return result == null ? alt : result;
+  }
```

Comment:
You use this method in more than one file. Is it better to refactor it to a new class?